cap/modules/fixtures/schemas/cms.json

```diff
@@ -19,6 +19,15 @@
             "description": "Please provide information about the main measurements of your analysis",
             "id": "main_measurements"
           },
+          "adl_schema": {
```

Comment:
I think we should move this in `basic_info` and have it as type `object` with properties:
```
{
   "file": { "type": "string", description: "Select an uploaded file" },
   "content": { "$ref":  "https://analysispreservation.cern.ch/schemas/deposits/records/adl-schema-v0.0.1.json" }
}
```